Crash report 4.3.2 - NPE starting service for comment

Low-risk fix.

```
Caused by: java.lang.NullPointerException
       at android.content.ComponentName.(ComponentName.java:77)
       at android.content.Intent.(Intent.java:4334)
       at org.wordpress.android.ui.reader.services.ReaderCommentService.startServiceForComment(ReaderCommentService.java:50)
       at org.wordpress.android.ui.notifications.NotificationsDetailListFragment.requestReaderContentForNote(NotificationsDetailListFragment.java:431)
       at org.wordpress.android.ui.notifications.NotificationsDetailListFragment.access$200(NotificationsDetailListFragment.java:52)
       at org.wordpress.android.ui.notifications.NotificationsDetailListFragment$LoadNoteBlocksTask.doInBackground(NotificationsDetailListFragment.java:263)
       at org.wordpress.android.ui.notifications.NotificationsDetailListFragment$LoadNoteBlocksTask.doInBackground(NotificationsDetailListFragment.java:255)
       at android.os.AsyncTask$2.call(AsyncTask.java:288)
       at java.util.concurrent.FutureTask.run(FutureTask.java:237)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
       at java.lang.Thread.run(Thread.java:841)
```

--- a/WordPress/src/main/java/org/wordpress/android/ui/reader/services/ReaderCommentService.java
+++ b/WordPress/src/main/java/org/wordpress/android/ui/reader/services/ReaderCommentService.java
@@ -38,6 +38,8 @@ public class ReaderCommentService extends Service {
     private static int mCurrentPage;
 
     public static void startService(Context context, long blogId, long postId, boolean requestNextPage) {
+        if (context == null) return;
+
         Intent intent = new Intent(context, ReaderCommentService.class);
         intent.putExtra(ARG_BLOG_ID, blogId);
         intent.putExtra(ARG_POST_ID, postId);
@@ -47,6 +49,8 @@ public class ReaderCommentService extends Service {
 
     // Requests comments until the passed commentId is found
     public static void startServiceForComment(Context context, long blogId, long postId, long commentId) {
+        if (context == null) return;
+
         Intent intent = new Intent(context, ReaderCommentService.class);
         intent.putExtra(ARG_BLOG_ID, blogId);
         intent.putExtra(ARG_POST_ID, postId);

